index.tsx

```diff
@@ -79,14 +80,16 @@ function MasonryList<T>(props: Props<T>): ReactElement {
       contentContainerStyle={contentContainerStyle}
       removeClippedSubviews={removeClippedSubviews}
       refreshControl={
-        <RefreshControl
-          refreshing={!!(refreshing || isRefreshing)}
-          onRefresh={() => {
-            setIsRefreshing(true);
-            onRefresh?.();
-            setIsRefreshing(false);
-          }}
-        />
+        props.refreshControl === false ? null : (
+          <RefreshControl
+            refreshing={!!(refreshing || isRefreshing)}
+            onRefresh={() => {
+              setIsRefreshing(true);
+              onRefresh?.();
+              setIsRefreshing(false);
+            }}
+          />
+        )
```

Comment:
Perhaps this would be cleaner?

(In response to https://github.com/hyochan/react-native-masonry-list/pull/39/files#r917482422)

```suggestion
        props.refreshControl ? (
          <RefreshControl
            refreshing={!!(refreshing || isRefreshing)}
            onRefresh={() => {
              setIsRefreshing(true);
              onRefresh?.();
              setIsRefreshing(false);
            }}
          />
        ) : null
```